RunSection/Actions/ActionLogSpace.h

```diff
@@ -0,0 +1,38 @@
+/////////////////////////////////////////////////////////////////////////
+// ActionLogSpace (RunSection module)
+// ------------------
+//
+// Molecular Spin Dynamics Software - developed by Claus Nielsen and Luca Gerhards.
+// (c) 2019 Quantum Biology and Computational Physics Group.
+// See LICENSE.txt for license information.
+/////////////////////////////////////////////////////////////////////////
+#ifndef MOD_RunSection_ActionLogSpace
+#define MOD_RunSection_ActionLogSpace
+
+#include "Action.h"
+#include <armadillo>
+#include <memory>
+
+namespace RunSection
+{
+    class ActionLogSpace : public Action
+    {
+    private:
+        //Data members
+        ActionScalar* actionScaler;
+        
+        std::shared_ptr<arma::rowvec> m_Points;
+        int m_Num;
```

Comment:
please try to go along with the same code styl such as the other two actions (the old ones, scalar and rotatevector)